app/utils/gitlab_manager.py

```
@@ -0,0 +1,124 @@
+                "actions": actions
+            })
+            print("Successfully committed all file changes")
+            return True
+        except Exception as e:
+            self.log_error("Error committing files", e)
+            return False
+
+    def create_merge_request(self, branch_name, merge_descripton):
+        """Creates a merge request from the feature branch to main."""
+        try:
+            mr = self.project.mergerequests.create({
```

Ruff (0.11.9)

Local variable `mr` is assigned to but never used

Remove assignment to unused variable `mr`

(F841)